pkg/assets/static/redirectpage.js

```diff
@@ -0,0 +1,53 @@
+const queryString = window.location.search;
+const urlParams = new URLSearchParams(queryString);
+const link = urlParams.get("link");
+const keyword = urlParams.get("keyword");
+const selectedId = urlParams.get("selectedId");
+const consoleUrl = "https://console.redhat.com/openshift/sandbox";
```

Comment:
We will probably need to move all specific URLs like console.redhat.com to a configuration so it can be configurable in different sandbox instances but we can keep it as is for now.

Reply:
or we could include it in the response of the `/api/v1/signup` endpoint - we should have these links available there, right?

Reply:
Yes, the configuration itself is exposed to the reg-page through our reg-service `/api/v1`. So UI doesn't have to get access to the configuration directly.

Reply:
I can use consoleUrl from the JSON data but, when it is not ready or when there is an error I would have get it from other config maybe